src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -787,23 +787,23 @@ public boolean accept(
                     this.currentIdx = currentIdx;
                 }
 
-                boolean isNewest = maxSentFrame == null
-                    || TimeUtils.rtpDiff(srcTs, maxSentFrame.srcTs) > 0;
+                boolean isNewerThanMostRecentFrame = !haveSentFrame()
+                    || RTPUtils.isNewerTimestampThan(srcTs, mostRecentSentFrame.srcTs);
 
-                boolean isNewerThanMostRecentKeyFrame = mrKeyFrame == null
-                    || TimeUtils.rtpDiff(srcTs, mrKeyFrame.srcTs) > 0;
+                boolean isNewerThanMostRecentKeyFrame = !haveSentKeyFrame()
+                    || RTPUtils.isNewerTimestampThan(srcTs, mostRecentSentKeyFrame.srcTs);
 
-                if (currentIdx > -1
+                if (!isSuspended()
                     // we haven't seen anything yet and this is an independent
                     // frame.
-                    && (maxSentFrame == null && sourceFrameDesc.isIndependent()
+                    && (!haveSentFrame() && sourceFrameDesc.isIndependent()
```

Comment:
i think both are correct, it just depends on what perspective you name the variable from...i was thinking of it from the point of the class itself, so it'd be "we" or "i" ("have i/we sent a frame yet?" -> `haveSentFrame`) but it'd also be correct to say "has it/this class sent a frame yet?" -> `hasSentFrame`.  not sure i feel strongly about one way vs the other.